extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -146,12 +150,18 @@ export class DoubleclickA4aEligibility {
       warnDeprecation('remote.html');
     }
     let experimentId;
-    const urlExperimentId = extractUrlExperimentId(win, element);
-    if (hasUSDRD || (useRemoteHtml &&
-                     !element.getAttribute('rtc-config') &&
-                     (!urlExperimentId ||
-                     URL_EXPERIMENT_MAPPING[urlExperimentId] !=
-                      DOUBLECLICK_EXPERIMENT_FEATURE.REMOTE_HTML_EXPERIMENT))) {
+    const urlExperimentId = extractUrlExperimentId(win, element) || '';
+    if (hasUSDRD && (!urlExperimentId ||
+          ![DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_EXPERIMENT,
+            DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_CONTROL].includes(
```

Comment:
shouldn't this be:  

if (hasUSDRD && urlExperimentId != DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_EXPERIMENT) return false;